src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -130,8 +136,8 @@ public HostOutputMessage HostOutputMessage
                     logCompletedCancellationToken = cancellationToken;
                 }
 
-                await NotifyCompleteAsync(fastItem, functionCompletedMessage.Arguments, exceptionInfo);
-                _resultsLogger?.LogFunctionResult(fastItem);
+                await NotifyCompleteAsync(instanceLogEntry, functionCompletedMessage.Arguments, exceptionInfo);
```

Comment:
>NotifyCompleteAsync [](start = 22, length = 19)

This is the "Stop Billing" event. An exception coming from the new call to InvokeExceptionFiltersAsync (which is feasible since it calls user code)  might skip this.  The call to InvokeExceptionFiltersAsync should probably be wrapped in a try/catch.